api/src/api/admin.js

```diff
@@ -166,17 +166,141 @@ router.post(
   }),
 );
 
+router.put(
```

Comment:
A few notes on this endpoint:
* Could we add a comment at the top saying this is for renaming categories?
* I think the `updatedResources` returned isn't what we expect - I also don't it's super necessary to return, so we can just get rid of that variable so that the code is:
```
await Resource.updateMany(
    { category: req.body.currentName },
    { $set: { 'category.$': req.body.newName } },
);

res.json({
    code: 200,
    message: `Successfully updated category ${id}`,
    success: true,
    result: updatedCategory
```
* I wonder if we can do a similar thing we did for renaming subcategories - so instead of passing the entire category object through `req.body` and passing that into `findByIdAndUpdate`, we can probably  just replace `req.body` with `{ $set: { 'name': req.body.newName } }`